week-1-ts-ecosystem/w01e01-typescript-basics/src/helpers/hasGivenAge.ts

```diff
@@ -0,0 +1,7 @@
+import { User } from "../types";
+
+function hasGivenAge(requiredAge: User["age"]) {
+  return (user: User) => user.age >= requiredAge;
+}
+
+export default hasGivenAge;
```

Comment:
Helpers to dobry pomysł, jednak "optymistycznie" sklasyfikowałbym to jako `validators`
bo zauważ, że każdy z nich odpowiada na pytanie - czy jakieś założenie jest spełnione.

unikaj `default` exportów, czasami mają one sens jednak w większości przypadków
polecam `named` exporty - żeby nie "zgubić" nazw. Zwłaszcza przy refaktoringu
Wtedy nazwa importu niewiele ma wspólnego z tym co importuje